### PR TITLE
Get all titles of the brick webpage.

### PR DESCRIPTION
**Pull request by jrdmose**

### CODE REVIEW

**Comment by jrdmose** [comment]
Closing it in order to submit the second part of the exercise

File: exercises.ipynb [hunk 1]
```diff
@@ -52,59 +49,80 @@
...
+    "def get_next_page_soup (currentsoup):\n",
+    "    \"\"\"Returns the url of next page\"\"\"\n",
+    "    #currentsoup is type sbeautifulsoup object\n",
+    "    #looks in the web page and returns next page url if exists, otherwise returns a blank string    \n",
+    "    url = currentsoup.select(\".next\")[0].a\n",
+    "    \n",
+    "    if url:\n",
+    "        return url[\"href\"]\n",
+    "    else:\n",
+    "        return \"\"\n",
```
Comment (nandanrao) [inline exercises.ipynb:61]: Why blank string?? This is what "None" is for!

File: exercises.ipynb [hunk 2]
```diff
@@ -52,59 +49,80 @@
    },
    "outputs": [],
    "source": [
+    "def get_next_page_soup (currentsoup):\n",
```
Comment (nandanrao) [inline exercises.ipynb:52]: I like the function though! Nice design!

File: exercises.ipynb [hunk 3]
```diff
@@ -52,59 +49,80 @@
...
-    "    # Use the \"get_titles\" function to extract the\n",
-    "    # titles from the BeautifulSoup object.\n",
-    "    #\n",
-    "    # Return the titles\n",
-    "    pass"
+    "    \"\"\"get all items in all the pages of the url\"\"\"\n",
+    "    \n",
+    "    bricks = []\n",
+    "    #the breaking condition is that we get a blank url which will be the case when we get to the last page of the website\n",
+    "    while url :\n",
```
Comment (nandanrao) [inline exercises.ipynb:87]: Nice and clean!

**Review by nandanrao — COMMENTED** [review]
Overall, very elegant!

**Pull request closed without merging**